src/lib/Scenes/Artwork/Components/ArtworkHistory.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Sans } from "@artsy/palette"
+import { ArtworkHistory_artwork } from "__generated__/ArtworkHistory_artwork.graphql"
+import { ReadMore } from "lib/Components/ReadMore"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface ArtworkHistoryProps {
+  artwork: ArtworkHistory_artwork
+}
+
+interface ArtworkHistoryState {
+  expand: boolean
+}
+
+export class ArtworkHistory extends React.Component<ArtworkHistoryProps, ArtworkHistoryState> {
+  state = { expand: false }
+
+  render() {
+    const sections = [
+      { title: "Provenance", value: this.props.artwork.provenance },
```

Comment:
#minor but you could destructure these earlier to avoid having to traverse multiple times:
```
{ provenance, exhibition_history, literature } = this.props.artwork
```